herbert_core/applications/multifit/_test/data/create_testdata_multifit_1.m

```diff
@@ -24,29 +24,29 @@
 x1=-10:2:90;
 x1=x1 + 0.1*rand(size(x1));
 y1=mftest_gauss_bkgd(x1,p1);
-e1=sqrt(y1);
+e1=y1;
```

Comment:
As noted previously, perhaps rewrite as
```matlab
[y1, e1] = noisify(y1, y1);
e1 = sqrt(e1);
...
```

Reply:
Noted. BTW, these tests are somewhat flakey; they are on my to-do list to overhaul as they are not automatically performed, and in the case of this function (for example) there is duplication of the same file as the one in herbert/_test/test_multifit.